### PR TITLE
XML Notepad version 2.9.0.8

### PR DESCRIPTION
**Pull request by lovettchris**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/145957)

### CODE REVIEW

File: manifests/m/Microsoft/XMLNotepad/2.9.0.8/Microsoft.XMLNotepad.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Created using wingetcreate 1.2.7.0
+# Created using wingetcreate 1.2.5.0
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
 
 PackageIdentifier: Microsoft.XMLNotepad
-PackageVersion: 2.9.0.5
+PackageVersion: 2.9.0.8
 DefaultLocale: en-US
 ManifestType: version
 ManifestVersion: 1.4.0
```
Comment (stephengillie) [inline manifests/m/Microsoft/XMLNotepad/2.9.0.8/Microsoft.XMLNotepad.yaml:8]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/m/Microsoft/XMLNotepad/2.9.0.8/Microsoft.XMLNotepad.installer.yaml [hunk 2]
```diff
@@ -15,16 +15,17 @@ InstallModes:
...
   InstallerType: msix
-  InstallerUrl: https://github.com/microsoft/XmlNotepad/releases/download/2.9.0.5/XmlNotepadPackage_2.9.0.5_AnyCPU.msixbundle
-  InstallerSha256: 7BE447DB659DB453B3DCC553FA807F015D8DEC3E250389FD000159D2AF678A08
-  SignatureSha256: 753B22B70F9C581F6CEF555497D485E6E127E2F9EA47358F4D900E9B52D51528
+  InstallerUrl: https://github.com/microsoft/XmlNotepad/releases/download/2.9.0.8/XmlNotepadPackage_2.9.0.8_AnyCPU.msixbundle
+  InstallerSha256: E978A89E0ED70814DA4031646F2B8449C31ACDAA4C50187C5A3EC330204A5DDA
+  SignatureSha256: 69277D98772D613E3C60219B8A8E48CF6E15266E4FBBDB37FBDEA780F4364522
   PackageFamilyName: 43906ChrisLovett.XmlNotepad_hndwmj480pefj
 ManifestType: installer
 ManifestVersion: 1.4.0
```
Comment (stephengillie) [inline manifests/m/Microsoft/XMLNotepad/2.9.0.8/Microsoft.XMLNotepad.installer.yaml:31]:
```suggestion
ManifestVersion: 1.6.0
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @lovettchris,

Most of our manifest fields are identical between versions, so usually just increasing the version number is enough to pass this check.

I wasn't able to suggest this for the file `Microsoft.XMLNotepad.locale.en-US.yaml`, but all 3 will need this identical change.

**Comment by lovettchris** [comment]
Ok, I ran `winget update wingetcreate` and a new `wingetcreate update` and pushed the changes, there were more than you proposed above.  I added `winget update wingetcreate` to my publish script so it should stay up to date now.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**